spec.md

```diff
@@ -78,10 +78,11 @@ regarding its semantic. Events are considered to be facts that have no given
 destination. Events are used to notify other systems that something has happened.
 
 #### Context
-A set of consistent metadata attributes included with the event about the
-occurrence that tools and developers can rely upon to better handle the event.
-These attributes describe the event and the structure of its data, include
-information about the originating system, and more.
+A set of metadata attributes included with the event that describe circumstances
+of the occurrence which are not specific to the occurrence. Tools and application
```

Comment:
The ID is definitely specific to the occurrence at least on some systems... I find this confusing.

Reply:
The ID is specific, but the circumstance of having an event ID is not.

Reply:
I'm still not getting it either. What does "the circumstance of having an ID" mean? Just that the concept of an `event-id` field is not specific? But having an `event-id` field contain a value that is unique to this event is somehow not specific to the occurrence? My understanding of the `event-id` field is that it *does* uniquely identify this occurrence.

Reply:
The original description seems to be more clear to me.  in addition to "... tools and developers can rely upon to better handle the event" , we may want to say "... tools, developers, serverless platforms can reply upon to better handle the event"

Reply:
@clemensv ^^^

Reply:
I condensed "tools and developers can rely upon to better handle" to "code can use"

Reply:
The event ID is context data. It's externally attached to the occurrence data for the purpose of embedding the occurrence data in an event. The event has uniqueness requirements for the ID. The best the occurrence data might give you is a timestamp of its local clock. 

It'd be great if anyone has a better term than "specific" to express the following two concepts:
* not part of the core data (occurrence information)
* describes the circumstances of the event being emitted (system scope, event type, etc)

Reply:
I wonder if this slight rewording would address some of the concerns:
- - -
As described in the **Event** definition, an Event contains two parts, the data representing the Occurrence and additional metadata that provides other circumstantial information about the Occurrence (e.g. information about the originating system). This additional metadata is referred to as **Context** data. Tools and application code can use this information to identify the relationship of Events to aspects of the system or to other Events.
- - -

Reply:
Since @clemensv is out, @djrosanova what do you think of the wording I've proposed?

@maplebed @deissnerk @cathyhongzhang @ultrasaurus could you guys live with it?

Reply:
Doug, LGTM. Thanks.